lbz/lambdas/response.py

```diff
@@ -0,0 +1,23 @@
+from typing import Any, Optional, TypedDict
+
+from lbz.lambdas.enums import LambdaResult
+
+
+class LambdaResponse(TypedDict, total=False):
+    result: str
+    data: Optional[Any]
+    message: Optional[str]
+    error_code: Optional[str]
+
+
+def lambda_ok_response(data: dict = None) -> LambdaResponse:
```

Comment:
It would be good to write tests for this function as well, as it contains more logic inside 😉